converter/solid_figures.py

```diff
@@ -15,15 +15,22 @@ class SolidFigure(ABC):
     position: tuple[float, float, float] = field(default_factory=lambda: [0., 0., 0.])
     rotation: tuple[float, float, float] = field(default_factory=lambda: [0., 0., 0.])
 
+    def expand(self, expansion: float) -> None:
+        """Expand figure by `expansion` in each dimension."""
 
-@dataclass(frozen=True)
+
+@dataclass(frozen=False)
 class SphereFigure(SolidFigure):
     """A sphere. Its size is defined by its radius."""
 
     radius: float = 1.
 
+    def expand(self, expansion: float) -> None:
+        """Expand figure by `expansion` in each dimension. Increases figures radius by `expansion`"""
+        self.radius += expansion/2
+
```

Comment:
The comment is inconsistent with the code. Why `expansion/2` ?

Reply:
If we increase by `expansion` the diameter will increase by `2*expansion`. Expansion on a sphere would make it expand twice as much compared to a box.